packages/libs/react-components/src/components/Typography/Heading.tsx

```diff
@@ -0,0 +1,24 @@
+import { BaseText } from './styles';
+
+import type { VariantProps } from '@stitches/react';
+import React, { FC } from 'react';
+
+export interface IHeadingProps {
+  as?: 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6';
+  font?: VariantProps<typeof BaseText>['font'];
+  bold?: VariantProps<typeof BaseText>['bold'];
```

Comment:
I think I would prefer a variantname instead of having to set font and bold props.
So all the variants in the styleguide should have a name. and we use those names as a variant.

If we keep it like this and the style of a variant in the stylebook changes, we need to go through the whole codebase to change a prop, no?

Reply:
I would keep the `as` prop.
That seperates the styling from the html

Reply:
@Isa is working on compiling a list of "preset" variants of typography that are being used on the docs website that will be created using these base components. I am using the as prop to style the component right now. Is there a reason you want to separate the styling from the html?

Reply:
mmm I think I get it. Sometimes you want the semantic element to be different from the style. I'll add an additional prop `variant` in addition to the `as` prop